### PR TITLE
BugFix & Modify readme.md

### PR DESCRIPTION
**Pull request by waittttting**

### CODE REVIEW

File: src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java [hunk 1]
```diff
@@ -88,7 +88,10 @@ public String visit(CallExpression call) {
 
     @Override
     public String visit(ValueLiteralExpression valueLiteral) {
-        if (valueLiteral.getOutputDataType() == DataTypes.DATE() || valueLiteral.getOutputDataType().toString().equals("DATE NOT NULL")) {
+        if (valueLiteral.getOutputDataType().getLogicalType().getTypeRoot().equals(DataTypes.TIMESTAMP().getLogicalType().getTypeRoot()) ||
```
Comment (hffariel) [inline src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java:91]: Use LogicalTypeRoot.TIMESTAMP instead

Comment (hffariel) [inline src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java:91]: Create a ref variable for valueLiteral.getOutputDataType().getLogicalType().getTypeRoot()

**Review by hffariel — CHANGES_REQUESTED** [review]

File: src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java [hunk 2]
```diff
@@ -88,7 +88,10 @@ public String visit(CallExpression call) {
 
     @Override
     public String visit(ValueLiteralExpression valueLiteral) {
-        if (valueLiteral.getOutputDataType() == DataTypes.DATE() || valueLiteral.getOutputDataType().toString().equals("DATE NOT NULL")) {
+        if (valueLiteral.getOutputDataType().getLogicalType().getTypeRoot().equals(DataTypes.TIMESTAMP().getLogicalType().getTypeRoot()) || 
+            valueLiteral.getOutputDataType().getLogicalType().getTypeRoot().equals(DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE().getLogicalType().getTypeRoot()) || 
+            valueLiteral.getOutputDataType().getLogicalType().getTypeRoot().equals(DataTypes.TIMESTAMP_WITH_TIME_ZONE().getLogicalType().getTypeRoot()) ||
+            valueLiteral.getOutputDataType().getLogicalType().getTypeRoot().equals(DataTypes.DATE().getLogicalType().getTypeRoot())) {
             return "'" + valueLiteral.toString() + "'";
```
Comment (hffariel) [inline src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java:95]: What if the valueLiteral's type is among [`VARCHAR`, `CHAR`, ...]

**Review by hffariel — CHANGES_REQUESTED** [review]

File: src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java (hunk 2 above)
Comment (waittttting) [reply src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java:97]: If the valueLiteral's type is among [VARCHAR, CHAR, ...]，The value of the expression is automatically enclosed in single quotes

**Pull request merged**